server/src/main/java/org/elasticsearch/health/HealthService.java

```diff
@@ -30,23 +30,31 @@ public HealthService(List<HealthIndicatorService> healthIndicatorServices) {
         this.healthIndicatorServices = healthIndicatorServices;
     }
 
-    public List<HealthComponentResult> getHealth() {
+    public List<HealthComponentResult> getHealth(String componentName, String indicatorName, boolean includeDetails) {
```

Comment:
would it make sense to rename `includeDetails` to `computeDetails` to indicate we'd like these not only to not be included but not calculated as much as possible?

Reply:
Makes sense. Done.